src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -19,6 +19,8 @@ import { processConfigEntities } from "../common/process-config-entities";
 import { createRowElement } from "../common/create-row-element";
 import { EntitiesCardConfig, EntitiesCardEntityConfig } from "./types";
 import { applyThemesOnElement } from "../../../common/dom/apply_themes_on_element";
+import { createHeaderFooterElement } from "../common/create-header-footer-element";
```

Comment:
Do we want to have 1 method for both headers and footers? Will there be elements that really better fit just a header or footer ?

Reply:
aesthetically, probably, but realistically, I doubt it.